guides/source/form_helpers.md

```diff
@@ -64,184 +64,227 @@ To create this form you will use `form_with` and the form builder object it yiel
-  <input type="text" name="article[title]" id="article_title" value="My Title" />
-  <textarea name="article[body]" id="article_body" cols="60" rows="10">
-    My Body
-  </textarea>
-  <input type="submit" name="commit" value="Update Article" data-disable-with="Update Article">
+<form action="/books" accept-charset="UTF-8" method="post">
+  <input type="hidden" name="authenticity_token" value="ChwHeyegcpAFDdBvXvDuvbfW7yCA3e8gvhyieai7DhG28C3akh-dyuv-IBittsjPrIjETlQQvQJ91T77QQ8xWA" autocomplete="off">
+  <div>
+    <label for="book_title">Title</label>
+    <input type="text" name="book[title]" id="book_title">
+  </div>
+  <div>
+    <label for="book_author">Author</label>
+    <input type="text" name="book[author]" id="book_author">
+  </div>
+  <input type="submit" name="commit" value="Create Book" data-disable-with="Create Book">
 </form>
 ```
 
-There are several things to notice here:
+Some important things to note when using `form_with` with a model object:
 
-* The form `action` is automatically filled with an appropriate value for `@article`.
-* The form fields are automatically filled with the corresponding values from `@article`.
-* The form field names are scoped with `article[...]`. This means that `params[:article]` will be a hash containing all these field's values. You can read more about the significance of input names in chapter [Understanding Parameter Naming Conventions](#understanding-parameter-naming-conventions) of this guide.
-* The submit button is automatically given an appropriate text value.
+* The form `action` is automatically filled with an appropriate value, `action="/books"`. If you were updating a book, it would be `action="/books/42"`.
+* The form field names are scoped with `book[...]`. This means that `params[:book]` will be a hash containing all these field's values. You can read more about the significance of input names in chapter [Form Input Naming Conventions and Params Hash](#form-input-naming-conventions-and-params-hash) of this guide.
+* The submit button is automatically given an appropriate text value, "Create Book" in this case.
 
-TIP: Conventionally your inputs will mirror model attributes. However, they don't have to! If there is other information you need you can include it in your form just as with attributes and access it via `params[:article][:my_nifty_non_attribute_input]`.
+TIP: Typically your form inputs will mirror model attributes. However, they don't have to. If there is other information you need you can include a field in your form and access it via `params[:book][:my_non_attribute_input]`.
 
-#### Composite primary key forms
+#### Composite Primary Key Forms
 
-Forms may also be built with composite primary key models. In this case, the form
-building syntax is the same with slightly different output.
+If you have a model with a [composite primary key](active_record_composite_primary_keys.html), forms building syntax is the same with slightly different output.
```

Comment:
I think this should be either:
forms building syntax **are** the same
or
**the** form building syntax is the same